redoctober.go

```diff
@@ -765,6 +772,11 @@ var indexHtml = []byte(`<!DOCTYPE html>
 					data.Labels[i] = data.Labels[i].trim();
 					if (data.Labels[i] == "") { data.Labels.splice(i, 1); }
 				}
+				data.Usages = data.Usages.split(',');
+				for(var i=0, l=data.Usages.length; i<l; i++){
+					data.Usages[i] = data.Usages[i].trim();
+					if (data.Usages[i] == "") { data.Usages.splice(i, 1); }
```

Comment:
Nobody is updating Red October to use rice at the moment. We switched cfssl to rice but haven't done it here.